src/panels/lovelace/editor/card-editor/hui-card-preview.ts

```diff
@@ -81,6 +85,10 @@ export class HuiCardPreview extends HTMLElement {
     this._cleanup();
     this._element = createCardElement(configValue);
 
+    if (this._element.tagName === "HUI-ERROR-CARD") {
```

Comment:
If we're already deriving it here, why even keep track of `this._error`  ? Can't we make it a getter that does this under the hood? 

```ts
private get _error() { return this._element && this._element.tagName === "HUI-ERROR-CARD"; }
```